src/Microsoft.DotNet.Build.Tasks.Feed/CopyBlobDirectory.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Framework;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using MSBuild = Microsoft.Build.Utilities;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace Microsoft.DotNet.Build.Tasks.Feed
+{
+    public sealed class CopyBlobDirectory : MSBuild.Task
+    {
+        [Required]
+        public string SourceBlobDirectory { get; set; }
+
+        [Required]
+        public string TargetBlobDirectory { get; set; }
+
+        [Required]
+        public string AccountKey { get; set; }
+
+        public bool Overwrite { get; set; }
+
+        public bool SkipCreateContainer { get; set; } = false;
+
+        public bool SkipIfMissing { get; set; } = true;
```

Comment:
Why is this set to true by default? I would expect this case to always be an error.

Reply:
@weshaggard FIxed.  This was intentional before when it was merging N folders from N input urls to a single output url, because not all input urls may have all the folders.